s4/str_conditional.rb

```diff
@@ -0,0 +1,21 @@
+# include?()
+text = "Snow white"
+print ".include?(\"S\") = "; p text.include?("S")
```

Comment:
Please put a single expression on each line.

Reply:
I’ll keep it in mind, I’ve done more to practice the difference between `p`, `puts`, and `print`.